src/core_plugins/kibana/public/visualize/editor/agg_params.js

```diff
@@ -81,15 +81,18 @@ uiModules
         // build collection of agg params html
         type.params.forEach(function (param, i) {
           let aggParam;
+          let fields;
           // if field param exists, compute allowed fields
           if (param.name === 'field') {
-            $aggParamEditorsScope.indexedFields = getIndexedFields(param);
+            fields = $aggParamEditorsScope.indexedFields = getIndexedFields(param);
+          } else if (param.type === 'field') {
+            fields = $aggParamEditorsScope[param.name] = getIndexedFields(param);
```

Comment:
What's the reason for switching away `indexedFields`? The usage of a `sortField` scope variable in `top_sort.html` really confused me until I got all the way to this change.